trieTools/trieToolsCommon/common.go

```diff
@@ -131,20 +134,22 @@ func CreatePruningStorer(flags ContextFlagsConfig, maxDBValue int) (storage.Stor
 	localDbConfig.FilePath = path.Join(flags.WorkingDir, flags.DbDir)
 
 	dbPath := path.Join(flags.WorkingDir, flags.DbDir)
-	args := &pruning.StorerArgs{
-		Identifier:                "",
-		ShardCoordinator:          testscommon.NewMultiShardsCoordinatorMock(1),
-		CacheConf:                 cacheConfig,
-		PathManager:               components.NewSimplePathManager(dbPath),
-		DbPath:                    "",
-		PersisterFactory:          factory.NewPersisterFactory(localDbConfig),
-		Notifier:                  notifier.NewManualEpochStartNotifier(),
-		OldDataCleanerProvider:    &testscommon.OldDataCleanerProviderStub{},
-		CustomDatabaseRemover:     disabled.NewDisabledCustomDatabaseRemover(),
-		MaxBatchSize:              45000,
-		NumOfEpochsToKeep:         uint32(maxDBValue) + 1,
-		NumOfActivePersisters:     uint32(maxDBValue) + 1,
-		StartingEpoch:             uint32(maxDBValue),
+	args := pruning.StorerArgs{
```

Comment:
the NewPersistersTracker is missing; could be added like this:

`epochsData := pruning.EpochArgs{
		NumOfEpochsToKeep:     uint32(maxDBValue) + 1,
		NumOfActivePersisters: uint32(maxDBValue) + 1,
		StartingEpoch:         uint32(maxDBValue),
	}

	args := pruning.StorerArgs{
		Identifier:                "",
		ShardCoordinator:          testscommon.NewMultiShardsCoordinatorMock(1),
		CacheConf:                 cacheConfig,
		PathManager:               components.NewSimplePathManager(dbPath),
		DbPath:                    "",
		PersisterFactory:          factory.NewPersisterFactory(localDbConfig),
		Notifier:                  notifier.NewManualEpochStartNotifier(),
		OldDataCleanerProvider:    &testscommon.OldDataCleanerProviderStub{},
		CustomDatabaseRemover:     disabled.NewDisabledCustomDatabaseRemover(),
		MaxBatchSize:              45000,
		EpochsData:                epochsData,
		PruningEnabled:            true,
		EnabledDbLookupExtensions: false,
		PersistersTracker:         pruning.NewPersistersTracker(epochsData),
	}`